src/main/java/Deadline.java

```diff
@@ -0,0 +1,25 @@
+public class Deadline extends Task {
+    private final String dueDate;
```

Comment:
The "final" keyword might be inappropriate here as i think dueDate is meant to be a variable that may be changed and not a constant.

Reply:
Yeah, that's something I'll need to account for. At the moment my Tasks are immutable, so the properties are all final as well. I'll refactor this in the future. :)